tests/scripts/check_requirements.py

```diff
@@ -56,7 +56,7 @@ def get_requirements_from_file(path):
         # A2A API internal imports
         "common", "task_manager", "agent"
     ],
-    "DEP002": ["psycopg2-binary", "lark", "transformers", "langchain-experimental", "lxml", "openpyxl", "onnxruntime", "pydantic_core"]
+    "DEP002": ["psycopg2-binary", "lark", "transformers", "langchain-experimental", "lxml", "openpyxl", "onnxruntime", "pydantic_core", "writer-sdk"]
```

Comment:
Is this being added because writer-sdk is imported under a different name, or because it's an optional dep?
Can we add a comment above explaining why it's there